.circleci/config.yml

```diff
@@ -36,8 +36,8 @@ jobs:
           sudo add-apt-repository -y ppa:beineri/opt-qt591-trusty
           sudo add-apt-repository -y ppa:george-edison55/cmake-3.x
           sudo apt-get -qy update
-          sudo apt-get install -y dbus xvfb curl cmake mesa-common-dev \
-              qt59base qt59graphicaleffects qt59quickcontrols2 qt59declarative qt59websockets
+          sudo apt-get install -y --force-yes dbus xvfb curl cmake mesa-common-dev \
```

Comment:
From [the apt-get linux man page](https://linux.die.net/man/8/apt-get):

> Force yes. This is a dangerous option that will cause apt-get to continue without prompting if it is doing something potentially harmful. It should not be used except in very special situations. Using --force-yes can potentially destroy your system!

Is this done to circumvent some problems after adding `qt59svg`?

Reply:
@madhavarshney this was done after failures in following ci run https://circleci.com/gh/MaxRis/react-native-desktop/341
The errors were:

```
Reading package lists...
W: GPG error: http://ppa.launchpad.net trusty InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 10C56D0DE9977759
W: GPG error: http://ppa.launchpad.net trusty InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 084ECFC5828AB726
...
WARNING: The following packages cannot be authenticated!
  cmake-data cmake qt59base qt59xmlpatterns qt59declarative
  qt59graphicaleffects qt59quickcontrols2 qt59svg qt59websockets
E: There are problems and -y was used without --force-yes
Exited with code 100
```
Maybe that was some temp issue with packages.
Probably, better to left this flag as is, since nothing very bad can happen on CircleCI's docker image, hopefully, and we don't want to experience such failures in the future :)

Reply:
I think it was a temp issue with packages.  For some reason GPG could not fetch the proper keys. 
 Relevant section of GPG errors:

```
gpg: keyring `/tmp/tmpjovwmzt3/secring.gpg' created
gpg: keyring `/tmp/tmpjovwmzt3/pubring.gpg' created
gpg: requesting key E9977759 from hkp server keyserver.ubuntu.com
gpgkeys: key C65D51784EDC19A871DBDBB710C56D0DE9977759 can't be retrieved
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
gpg: keyring `/tmp/tmpvtm636tq/secring.gpg' created
gpg: keyring `/tmp/tmpvtm636tq/pubring.gpg' created
gpg: requesting key 828AB726 from hkp server keyserver.ubuntu.com
gpgkeys: key B118CD3C377DF930EDD06C67084ECFC5828AB726 can't be retrieved
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
```

Either way, I think the best idea would be to create a custom docker image with CMake, Qt, and maybe Node.js preinstalled, as that's the [recommended way to do persistent apt-get installs](https://discuss.circleci.com/t/caching-apt-get-packages/9076/3) and would make sure any such issues don't appear.

Reply:
@MaxRis - Just [hit this issue](https://circleci.com/gh/status-im/react-native-desktop/581?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) in #279.  `--force-yes` doesn't seem to necessarily solve this issue because in that PR `apt-add-repository` fails with a non-zero exit code.  This seems to be a [widespread problem/known issue](https://github.com/docker-library/mysql/issues/263#issuecomment-354025886) with GPG keyservers.  As I mentioned above, a prebuilt docker image would be best.